frontend/src/metabase/app.js

```diff
@@ -68,6 +70,9 @@ function _init(reducers, getRoutes, callback) {
 
   initializeEmbedding(store);
 
+  // DO NOT RELEASE THIS IN PRODUCTION
+  const enableRRWeb = true;
```

Comment:
Can we make it an env to prevent releasing this?

Reply:
yup, that’s the idea, I’ll definitely rewrite this line before making a real PR

Reply:
I wonder if we can use something like our enterprise plugin system to avoid even shipping the tracking code at all in real builds.

Reply:
it would probably be for the best, because it the script is there we are probably required to update our Terms of service. Imagine if someone randomly found it 😬 

one solution that comes to mind is to simply add the rrweb script to codebase (wouldn’t really affect the bundle size) and conditionally import it.